### PR TITLE
Remove profile

### PR DESCRIPTION
**Pull request by ccjjmartin**
## Pending:
- [x] Needs to overwrite scaffolding coming from core on settings.php so that we can setup `../config/default` as the directory for config.  Looks like this used to do it in the distro, I don't see anything in composer.json that would scaffold this though so it may be broken: https://github.com/fourkitchens/sous-drupal-distro/blob/main/assets/scaffold/default/additions-default.settings.txt scaffolding the 

## Changes:
- Adds config to the project
- Removes drupal/console
- Removes fourkitchens/sous-drupal-distro
- Updates the node version
- Updates the Emulsify CLI

### CODE REVIEW

**Review by callinmullaney — CHANGES_REQUESTED** [review]
@ccjjmartin See my nvm code comment. Also, while I am able to build a local project successfully with this branch I believe the `uuid` and 
```_core:
  default_config_hash:
```
stuff should be removed from the config files. These are added to the config after the first export post-install and should be unique per-site.

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-lts/gallium
+16.13
```
[the remark this answers is not in the document]
Comment (ccjjmartin) [reply .nvmrc]: @callinmullaney Reverted

**Comment by ccjjmartin** [comment]
@callinmullaney As discussed the core key shouldn't be in issue anymore so I think we can leave it.  This should be ready for you to pull into your branch.

**Review by callinmullaney — APPROVED** [review]
This has been a long road @ccjjmartin but I think we made it. :shipit:

**Pull request merged**